spec/classes/puppet_agent_install_spec.rb

```diff
@@ -8,7 +8,12 @@
     additional_facts = {:rubysitedir => '/opt/puppetlabs/puppet/lib/ruby/site_ruby/2.1.0'}
   end
 
-  let :common_centos_facts do on_supported_os['centos-6-x86_64'].merge({
+on_supported_os.each do |os, os_facts|
+  next if not os == "centos-6-x86_64"
```

Comment:
Yes. I'm adding a helper that evaluates an environment variable that can be set to reduce the set of platforms tested.